fastapi_pagination/ext/cassandra.py

```diff
@@ -0,0 +1,39 @@
+from typing import Any, Dict, Mapping, Optional, Type, TypeVar
+
+from cassandra.cluster import SimpleStatement
+from cassandra.cqlengine import connection
+from cassandra.cqlengine.models import Model
+
+from ..api import create_page
+from ..bases import AbstractPage
+from ..utils import TParams, verify_params
+
+T = TypeVar("T", bound=Mapping[str, Any])
+
+
+def paginate_with_cursor(
+    model: Type[Model],
+    query_filter: Optional[Dict[Any, Any]] = None,
+    params: Optional[TParams] = None,
+    **kwargs: Any,
```

Comment:
`kwargs` is not used, we can remove them.

Reply:
removed